keycloak/themes/haven/email/text/email-verification.ftl

```diff
@@ -0,0 +1,308 @@
+<!doctype html>

```

Comment:
I think the version of the template under the /text directory should be a plain text version of the email with no html markup.

The emails that keycloak sends are multi-part with both html and plain text parts, and the email client will choose which one to render. Some people have email clients configured to ignore html email, in those cases the plain text version will be rendered.

The copy and the links should be the same for both versions of the email.